Firmware/mmu.cpp

```diff
@@ -829,7 +839,15 @@ void mmu_M600_wait_and_beep() {
 		KEEPALIVE_STATE(PAUSED_FOR_USER);
 
 		int counterBeep = 0;
-		lcd_display_message_fullscreen_P(_i("Remove old filament and press the knob to start loading new filament."));
+		if (mmu_E_jam_detected)
+		{
+			lcd_display_message_fullscreen_P(_i("Possible extruder jam detected (IR=0). Fix problem and press knob."));
```

Comment:
Please add ` ////c=20 r=yy` behind `"));" to document the need rows on the LCD screen.
Also don't forget that translations are often longer than English.